js/app.js

```diff
@@ -81,10 +81,30 @@ TextApp.prototype.setTheme = function() {
   this.editor_.setTheme(theme);
 };
 
+/**
+ * remove the editor so it can be reinitalised
+ */
+TextApp.prototype.removeEditor = function() {
+  const editor = $('#editor')[0];
+  while(editor.firstElementChild !== null) {
+    editor.firstElementChild.remove();
+  }
+}
+
 /**
  * Called when all the services have started and settings are loaded.
  */
 TextApp.prototype.onSettingsReady_ = function() {
+  if (this.settings_.get('accessibilitymode')) {
+    this.removeEditor();
+    this.editor_ = new EditorTextArea($('#editor')[0], this.settings_);
```

Comment:
since we need to grab the editor from the DOM here I'd pass it as an arg to removeEditor rather than have to scan the DOM twice (same below)

Reply:
good idea!